src/components/v2/shared/DistributionSplitsSection/DistributionSplitCard.tsx

```diff
@@ -62,6 +75,11 @@ export default function DistributionSplitCard({
   const distributionLimitIsInfinite =
     !distributionLimit || parseWad(distributionLimit).eq(MAX_DISTRIBUTION_LIMIT)
 
+  // If percentage has greater than 2 dp it will be rounded in the UI
+  const percentIsRounded =
+    split.percent !== SPLITS_TOTAL_PERCENT &&
+    (split.percent / SPLITS_TOTAL_PERCENT).toString().split('.')[1].length > 4
```

Comment:
`split.percent / SPLITS_TOTAL_PERCENT` get an amount smaller than what is shown on the UI - e.g a split of 206500000 is shown on the UI as 20.65, however this calc will bring it to `0.2065`. This might be what you intended though

Reply:
Yeah it is, that's why it's `.length > 4` and not `>2` 😅